src/state/component-repository/components/reducer.js

```diff
@@ -297,6 +300,75 @@ const progressStatus = (state = 0, action = {}) => {
   }
 };
 
+const initialConflictsModalState = {
```

Comment:
Please remove "conflicts modal" with "install conflicts" (or whatever suits it better) within state. "Modal" is UI-related, not logic-related.

Consider this comment for all the occurrences of "conflicts modal" in the state files.